Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Set the base image to Python
+FROM python:2.7.9
+MAINTAINER code040@intelworks.com
```

Comment:
maybe use our default email here for consistency? https://github.com/Intelworks/OpenTAXII/blob/master/setup.py#L19